homework/src/index.js

```diff
@@ -30,18 +32,113 @@
     return elem;
   }
 
+  const root = document.getElementById('root');
+  const header = createAndAppend('header', root, { id: 'header' });
+  const h3 = createAndAppend('h3', header);
+  h3.innerText = 'HYF Repository';
+  const mainDiv = createAndAppend('div', root, { id: 'main' });
+  const div1 = createAndAppend('div', mainDiv, { id: 'div1' });
+  const div2 = createAndAppend('div', mainDiv, { id: 'div2' });
+  const ul = createAndAppend('ul', div1, { id: 'list-container' });
+  const Contributions = createAndAppend('h3', div2)
+  Contributions.innerText = 'Contributions';
+  const ul2 = createAndAppend('ul', div2, { id: 'list-container' });
```

Comment:
1. Lines 35-45 are executed as soon at `index.js` is loaded. It is best to start execution when the page is fully loaded, i.e., using the `window.onload` event handler. Currently this is set up to run the function `main()`. Therefore you should move these lines to the start of function `main()` just before the `fetchJSON()` call.

2. Avoid making names unique by using a number suffix (`div1`, `div2`). Spend a bit more effort in coming up with fitting names: what are these `div`-s used for? The answer should be reflected in their names. For instance: `'leftHandContainer` and `rightHandContainer`, or `repoDiv` and `contributorDiv` etc.

3. Remember that all variable and function names should be in camelCase: `Contributions` -> `contributions`.